src/components/layouts/layout.css

```diff
@@ -1,13 +1,20 @@
 body {
   margin: 0px;
   padding: 0px;
+  overflow-x: hidden;
 }
 
 /* Class definition */
 
 .onHoverDisplay {
+  height: 100%;
+  width: 100%;
   display: none;
 }
+.onHoverHide {
```

Comment:
These classes are usually used together, yes? Maybe DRY it:

```
.onHoverDisplay, .onHoverHide {
  height: 100%;
 width: 100%;
}

.onHoverDisplay {
  display: none;
}
```

Better yet, consider refactoring height & width into a separate class (e.g. `fullHeightWidth`), since it's not obvious to me why hover display classes should have anything to do with setting the hieght/width?

Reply:
I refactor it like that

In `styledComponents`
```
muiGridWithOnHoverText: {
    '& > .onHoverDisplay': {
      display: 'none',
      height: '100%',
      width: '100%',
    },
    '& > .onHoverHide': {
      display: 'flex',
      height: '100%',
      width: '100%',
    },
    '&:hover > .onHoverDisplay': { display: 'flex' },
    '&:hover > .onHoverHide': { display: 'none' },
  },
```

### A use case

```
<Grid className={muiGridWithOnHoverText}>
  <Grid className="onHoverHide">
    Text to display by default
  </Grid>
  <Grid className="onHoverDisplay">
    Text to display on hover
  </Grid>
</Grid>
```